### ISSUE TITLE
Unclear error message: "failed to compile project"

### ISSUE THREAD

**Issue by indianaj0nas** [orta/vscode-playdate#26]
Getting the following error when I attempt to Run App in Playdate Simulator from the VS Code command (accessed via cmd + shift + P).

```
vscode-playdate: Failed to compile project.

Source: Playdate (Extension)
```
I'm clueless as to what to do from that. The paths look correct in the settings.json.

I had it working last night but after updating the Playdate SDK from 2.0.1 to 2.0.3 I'm just getting this error instead.

**Comment by indianaj0nas** [comment]
I found the fix! I had not opened my Playdate project as an actual VS Code project. It was in some kind of undefined workspace. It was easy to fix, I just had to go to File > New Window and then click Open and locate the project folder again on VS Code's welcome screen.

Might be helpful to add a warning for that? Maybe it's so obvious that it doesn't seem necessary, I'm not very savvy with this program.
